tests/python/PycodestyleBearTest.py

```diff
@@ -48,7 +48,7 @@ def hello():
     invalid_files=(long_line,)
 )
 
-PycodestyleBearLineLengthTest = verify_local_bear(
+PycodestyleCustomBearLineLengthTest = verify_local_bear(
```

Comment:
The name of the bear is `PycodestyleBear`, this kinda messes the testname up a bit :sweat_smile: 
What about `PycodestyleBearLineLengthSettingTest`, because you are using the `max_line_length` setting?

Reply:
Yeah, sounds good :thumbsup:
I'll make the change.